app/src/features/surveys/view/survey-animals/TelemetryDeviceForm.tsx

```diff
@@ -68,25 +123,56 @@ const TelemetryDeviceForm = () => {
         <Grid item xs={6}>
           <TelemetrySelectField
             label="Device Manufacturer"
-            name={'device_make'}
+            name={`${index}.device_make`}
             id="manufacturer"
             fetchData={api.devices.getCollarVendors}
           />
         </Grid>
         <Grid item xs={6}>
-          <CustomTextField label="Device Model" name={'device_model'} />
-        </Grid>
-        <Grid item xs={6}>
-          <SingleDateField name={'attachment_start'} required={true} label={'Attachment Start'} />
+          <CustomTextField label="Device Model" name={`${index}.device_model`} />
         </Grid>
+      </Grid>
+      <Box marginTop={'12px'}>
+        <Paper sx={{ padding: '12px' }}>
+          <Typography marginLeft={'6px'} marginBottom={'12px'}>
+            Deployments
+          </Typography>
+          {<DeploymentFormSection index={index} deployments={values[index].deployments ?? []} />}
+        </Paper>
         {Object.entries(bctwErrors).length > 0 && (
           <Grid item xs={12}>
-            {Object.values(bctwErrors).map((bctwError) => (
-              <FormHelperText error={true}>{bctwError}</FormHelperText>
+            {Object.values(bctwErrors).map((a, i) => (
+              <FormHelperText key={`form-error-text-${i}`} error={true}>
+                {a}
+              </FormHelperText>
             ))}
           </Grid>
         )}
-      </Grid>
+      </Box>
+    </>
+  );
+};
+
+interface ITelemetryDeviceFormProps {
+  mode: TelemetryDeviceFormMode;
+}
+
+const TelemetryDeviceForm = ({ mode }: ITelemetryDeviceFormProps) => {
+  const { values } = useFormikContext<IAnimalTelemetryDevice[]>();
+
+  return (
+    <Form>
+      <>
+        {values.map((a, i) => (
+          <Box key={`device-form-section-${i}`} marginTop={'12px'}>
+            <Typography marginLeft={'12px'} marginBottom={'12px'}>
```

Comment:
Nit: prefer using the `sx` prop, which uses global units based on the app's theme:

```
// 3 units => 4px * 3 = 12px
<Typography sx={{ ml: 3, mb: 3 }} />
```